.travis.yml

```diff
@@ -36,36 +41,39 @@ install:
     - conda info -a
 
     # Install dependencies and enter test environment
-    - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION "numpy<1.13" scipy future nomkl sphinx sphinx_rtd_theme pytest pytest-pep8 pytest-cov
-    - source activate test-environment
-
-    # Some packages which are only on PyPI, not on conda
-    - pip install pyfftw pywavelets travis-sphinx
+    - conda create -n testenv python=$TRAVIS_PYTHON_VERSION nomkl "numpy=$NUMPY_VERSION" scipy future pywavelets scikit-image sphinx sphinx_rtd_theme pytest pytest-pep8 pytest-cov
```

Comment:
Perhaps only install sphinx when building docs? e.g. after the tests

Reply:
True.

For some odd reason the creation of the initial testenv takes forever (like 2 minutes). Also weird that it downloads mkl, but that doesn't really add anything. Perhaps it's just conda that's slow.

Reply:
I'll try using pip for the dependencies that do not require compile steps. Maybe that's faster.